pkg/webmiddleware/log.go

```diff
@@ -47,6 +51,12 @@ func Log(logger log.Interface) MiddlewareFunc {
 			r = r.WithContext(log.NewContext(ctx, requestLogger))
 			metrics := httpsnoop.CaptureMetrics(next, w, r)
 
+			if metrics.Code < 400 {
+				if _, ignore := ignorePaths[r.URL.Path]; ignore {
```

Comment:
Using a map here instead of going through the array of the paths for every HTTP request.

Maybe could rewrite as `ignorePaths map[string]bool` and `if metrics.Code < 400 && ignorePaths[r.URL.Path]` ? Seems more of a stylistic choice.

Reply:
I'm fine with this.